02-peer-protocol.md

```diff
@@ -488,10 +689,13 @@ A sending node:
   - MAY send a `shutdown` before a `funding_locked`, i.e. before the funding transaction has reached `minimum_depth`.
   - if there are updates pending on the receiving node's commitment transaction:
     - MUST NOT send a `shutdown`.
+  - if there is an ongoing splice:
+    - MUST NOT send a `shutdown`.
```

Comment:
this can be moved as required of quiescence? Splice required that the channel is in quiet mode so this propriety will be already satisfied

Sounds more natural to me

Reply:
Yes I agree

Reply:
There may be a case where we still want something more sophisticated around `shutdown`, `stfu`, and `splice` down the road here. Long term the goal is to make channel closes also be splices so multiple can be merged together as well as closing directly into a channel open.

More discussion is needed to solve the complexities around this 🤔

Reply:
> this can be moved as required of quiescence? Splice required that the channel is in quiet mode so this propriety will be already satisfied
> 
> Sounds more natural to me

Ah there is a misunderstanding here. `shutdown` is not allowed if there is a splice awaiting confirmation which can happen outside of `stfu`.

Perhaps this can be made clearer by specifying a "pending splice" instead of ongoing. I will change that.